package.json

```diff
@@ -13,10 +13,27 @@
   },
   "license": "MIT",
   "peerDependencies": {
+    "@swc/core": "^1.0.0",
+    "@types/content-type": "^1.0.0",
```

Comment:
are these type packages still required after https://github.com/seamapi/nextjs-server-modules/pull/35?

Reply:
Next was printing warnings about swc, so I had to add it back. It's a good question about the type packages but I think it's safer to keep them for now. If they are really not needed then we can remove them in the future as a patch version.